### PR TITLE
Vue 3

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app-data/_nav.js
Comment (randyhoulahan) [inline app-data/_nav.js:1]: this in by accident?

File: api/km-storage.js
Comment (rqk2k) [inline api/km-storage.js:1]: Should use `encodeUriComponent` on `await useAPIFetch(serviceUrl.documentUrl(identifier)` as it was used on line 6?

**Review by randyhoulahan — APPROVED** [review]

File: api/km-storage.js
Comment (blaisefonseca) [reply api/km-storage.js:1]: `serviceUrl.documentUrl` is a function which takes care of encodeUriComponent

File: app-data/_nav.js
Comment (blaisefonseca) [reply app-data/_nav.js:1]: deleted

**Review by rqk2k — APPROVED** [review]

**Pull request merged**